runtime/wasm/src/to_from/external.rs

```diff
@@ -390,6 +390,19 @@ impl FromAscObj<AscUnresolvedContractCall> for UnresolvedContractCall {
             contract_name: heap.asc_get(asc_call.contract_name),
             contract_address: heap.asc_get(asc_call.contract_address),
             function_name: heap.asc_get(asc_call.function_name),
+            function_signature: None,
+            function_args: heap.asc_get(asc_call.function_args),
+        }
+    }
+}
+
+impl FromAscObj<AscUnresolvedContractCall_0_0_4> for UnresolvedContractCall {
```

Comment:
Ok the `FromAscObj` impl difference for version 0.0.4 is clear here, you can disregard my question above.  Maybe it's worth including a brief comment explaining this situation on this impl or the ETHEREUM_CALL_FUNC_INDEX where it is used?

Reply:
Yep, I've added a comment right where `ETHEREUM_CAL_FUNC_INDEX` is handled.